crates/oxc_formatter/src/service/oxfmtrc.rs

```diff
@@ -351,13 +337,9 @@ impl Oxfmtrc {
             };
         }
 
+        // Not yet supported options:
         // [Prettier] experimentalOperatorPosition: "start" | "end"
-        if let Some(position) = self.experimental_operator_position {
-            options.experimental_operator_position = match position {
-                OperatorPositionConfig::Start => OperatorPosition::Start,
-                OperatorPositionConfig::End => OperatorPosition::End,
-            };
-        }
+        // [Prettier] experimentalTernaries: boolean
```

Comment:
How about returning an error directly here instead? Then, we should disable generating the schema for this option

Reply:
After taking another look, I realized it wasn't as bad as I thought. 🙂 => #16088